helpers/src/dynamodb.rs

```diff
@@ -183,6 +183,56 @@ macro_rules! val {
 	);
 }
 
+#[macro_export]
+/// Create a **HashMap** from a list of key-value pairs
```

Comment:
I notice you leave your documentation after the `#[macro_export]` here, but before in line 221. Although both are totally valid, it'd be nice to stick to one style :smile:

Reply:
Good point, I've changed it!